booking/templates/ticket.html

```diff
@@ -0,0 +1,409 @@
+{% load static %}
+{% load i18n %}
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="icon" type="image/icon" href="{% static 'logo/favicon.png' %}" />
+  <title>e-Ticket</title>
+  <style>
```

Comment:
move code css vào file internal em nha

Reply:
em thử move ra file ngoài r nma do đây là in ra pdf nên là khi nó hiển thị ra thì ko áp dụng được phần css đó vào được nên em phải để lại vào trong ạ